lib/lutaml/xsd/xsd.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Lutaml
+  module Xsd
+    class Error < StandardError; end
+
+    module_function
+
+    def parse(xsd)
+      Lutaml::Model::Config.xml_adapter_type = :nokogiri
```

Comment:
We do not need to set the adapter inside every method call this can be done outside the method.